docs/02-admin/02-configuration/04-postgresql.md

```diff
@@ -48,14 +48,20 @@ max_parallel_maintenance_workers = 4
 # You should *not* increase this value more than max_worker_processes
 max_parallel_workers = 16
 
-# Write ahead log sizes (unless you expect to write more than 1GB/hour of data in the DB)
-max_wal_size = 8GB
-min_wal_size = 2GB
+# Boost transaction speeds and reduce I/O wait for writes (with the risk of losing un-flushed data in case of a crash)
+# If you do not want to take that risk, keep it to: "on".
+synchronous_commit = off
 
 # Group write commits to combine multiple transactions by a single flush (this is a time delay in μs)
 commit_delay = 300
 
+# Increase the checkpoint timeout (time between two checkpoints) to reduce the disk I/O
+# This will significantly reduce the disk I/O and speed-up the write times to disk. The only downside is time needed for crash recovery.
+checkpoint_timeout = 40min
 checkpoint_completion_target = 0.9
+# Write ahead log sizes (so the WAL file can contain around 1 hour of data)
+max_wal_size = 30GB
```

Comment:
I think 40 min and 30 GB is abit much for the default...

Reply:
Nope. 40 min isn't for sure, this is very much recommended for every write database queries, we have with mbin. And 30gb maybe, but actually not for big instances.

See also https://www.crunchydata.com/blog/tuning-your-postgres-database-for-high-write-loads


The wall size was just too small even on my instance together with the checkpoint interval. My server was therefore constantly busy writing checkpoints. This is bad. I increased the max Wal size to be big enough, that the checkpoint will be triggered by the timeout.

Reply:
I believe 30GB is fine. It doesn't matter too much, the interval will trigger the checkpoint.

Even my small instance benefit of these values a lot as you can see in the grafana dashboard.

Reply:
Lets put it this wqy: people hosting from VPSes in the cloud will not have 30GB to spare. I dont on Gehirneimer for example.
Yes your instance might be smaller, but you have a gargantous server behind it compared to most servers on the fediverse 😁
If you say that it cannot cause problems, then I'll approve, but J am not sure 😅

Reply:
The wal file is stored in `$PGDATA/pg_wal`. I can decrease the wal size. But.. my database alone (which is stored in the $PGDATA is 48GB already..

Reply:
I decrease it  to 10GB, is that OK?

Reply:
Yeah that is fine. My DB is at about 22 GB I believe